src/main/java/com/amazon/opendistroforelasticsearch/jobscheduler/scheduler/JobScheduler.java

```diff
@@ -147,6 +147,8 @@ boolean reschedule(ScheduledJobParameter jobParameter, JobSchedulingInfo jobInfo
         }
         Duration duration = Duration.between(this.clock.instant(), nextExecutionTime);
 
+        jobInfo.setExpectedExecutionTime(nextExecutionTime);
```

Comment:
There are no changes to the variable `nextExecutionTime` below, so I moved the line a little bit above.